packages/viz/test/styles/colors.test.js

```diff
@@ -8,18 +8,31 @@ describe('colors', () => {
     expect(_.toLower(colors.bolusMicro)).to.equal('var(--bolus-micro)');
     expect(_.toLower(colors.bolusManual)).to.equal('var(--bolus-manual)');
     expect(_.toLower(colors.basal)).to.equal('#19a0d7');
+    expect(_.toLower(colors.basalHeader)).to.equal('#DCF1F9');
+    expect(_.toLower(colors.basalAutomated)).to.equal('#00D3E6');
     expect(_.toLower(colors.basalManual)).to.equal('#a8c8d4');
     expect(_.toLower(colors.statDark)).to.equal('#27385b');
     expect(_.toLower(colors.statDefault)).to.equal('#727375');
     expect(_.toLower(colors.statDisabled)).to.equal('#e7e9ee');
-    expect(_.toLower(colors.veryLow)).to.equal('var(--bg-very-low)');
-    expect(_.toLower(colors.low)).to.equal('var(--bg-low)');
-    expect(_.toLower(colors.target)).to.equal('var(--bg-target)');
-    expect(_.toLower(colors.high)).to.equal('var(--bg-high)');
-    expect(_.toLower(colors.veryHigh)).to.equal('var(--bg-very-high)');
+    expect(_.toLower(colors.veryLow)).to.equal('#DA3A1B');
+    expect(_.toLower(colors.low)).to.equal('#E98D7C');
+    expect(_.toLower(colors.target)).to.equal('#9FCC93');
+    expect(_.toLower(colors.high)).to.equal('#F9D83E');
+    expect(_.toLower(colors.veryHigh)).to.equal('##FFA700');
```

Comment:
```suggestion
    expect(_.toLower(colors.veryHigh)).to.equal('#FFA700');
```

Reply:
let me fix the tests, I did not looked at the toLower/toUpper and the way color codes are written